src/parquet/file/reader.h

```diff
@@ -34,6 +34,8 @@ struct RowGroupStatistics {
   int64_t num_values;
   int64_t null_count;
   int64_t distinct_count;
+  std::string min;
+  std::string max;
```

Comment:
Sounds good. Unit testing this might be too onerous since we cannot write fully-formed files yet. I hope to find some time to build out the write path later this month.